package.json

```diff
@@ -66,69 +66,69 @@
     "babel-core": "7.0.0-bridge.0",
     "babel-eslint": "10.1.0",
     "babel-jest": "26.6.3",
-    "babel-loader": "8.2.2",
+    "babel-loader": "8.2.3",
     "babel-plugin-module-resolver": "4.1.0",
     "babel-plugin-react-remove-properties": "0.3.0",
-    "babel-plugin-styled-components": "1.13.2",
+    "babel-plugin-styled-components": "1.13.3",
     "chalk": "4.1.2",
     "commander": "7.2.0",
-    "core-js": "3.17.3",
+    "core-js": "3.19.3",
     "coveralls": "3.1.1",
     "enhanced-resolve": "4.5.0",
-    "envalid": "7.2.1",
+    "envalid": "7.2.2",
     "eslint": "7.32.0",
     "eslint-config-prettier": "8.3.0",
     "eslint-loader": "4.0.2",
     "eslint-plugin-garden-local": "file:./utils/eslint",
-    "eslint-plugin-jest": "24.4.0",
-    "eslint-plugin-jsx-a11y": "6.4.1",
+    "eslint-plugin-jest": "24.7.0",
+    "eslint-plugin-jsx-a11y": "6.5.1",
     "eslint-plugin-node": "11.1.0",
     "eslint-plugin-notice": "0.9.10",
-    "eslint-plugin-react": "7.25.1",
-    "eslint-plugin-react-hooks": "4.2.0",
+    "eslint-plugin-react": "7.27.1",
+    "eslint-plugin-react-hooks": "4.3.0",
     "execa": "5.1.1",
-    "fork-ts-checker-webpack-plugin": "6.3.3",
+    "fork-ts-checker-webpack-plugin": "6.5.0",
     "fuzzy": "0.1.3",
     "github-markdown-css": "4.0.0",
-    "glob": "7.1.7",
-    "husky": "7.0.2",
+    "glob": "7.2.0",
+    "husky": "7.0.4",
     "identity-obj-proxy": "3.0.0",
-    "inquirer": "8.1.3",
+    "inquirer": "8.2.0",
     "inquirer-autocomplete-prompt": "1.4.0",
     "jest": "26.6.3",
-    "jest-styled-components": "7.0.5",
+    "jest-styled-components": "7.0.8",
     "lerna": "4.0.0",
     "lint-staged": "10.5.4",
-    "markdownlint-cli": "0.28.1",
+    "markdownlint-cli": "0.30.0",
     "micromatch": "4.0.4",
     "mockdate": "3.0.5",
     "ora": "5.4.1",
     "pluralize": "8.0.0",
-    "prettier": "2.4.0",
+    "prettier": "2.5.1",
     "prettier-package-json": "2.6.0",
-    "qs": "6.10.1",
+    "qs": "6.10.2",
     "react": "17.0.2",
     "react-dom": "17.0.2",
     "react-is": "17.0.2",
     "react-test-renderer": "17.0.2",
     "regenerator-runtime": "0.13.9",
     "resolve-from": "5.0.0",
-    "rollup": "2.56.3",
+    "rollup": "2.61.1",
     "rollup-plugin-analyzer": "4.0.0",
     "rollup-plugin-babel": "4.4.0",
     "rollup-plugin-cleanup": "3.2.1",
     "rollup-plugin-delete": "2.0.0",
-    "rollup-plugin-license": "2.5.0",
+    "rollup-plugin-license": "2.6.0",
     "rollup-plugin-size-snapshot": "0.12.0",
-    "rollup-plugin-typescript2": "0.30.0",
-    "styled-components": "5.3.1",
+    "rollup-plugin-typescript2": "0.31.1",
+    "styled-components": "5.3.3",
     "stylelint": "13.13.1",
     "stylelint-config-styled-components": "0.1.1",
     "stylelint-order": "4.1.0",
     "stylelint-processor-styled-components": "1.10.0",
     "temp": "0.9.4",
     "ts-jest": "26.5.6",
-    "typescript": "3.9.10",
-    "webpack": "5.58.0"
+    "typescript": "4.3.5",
```

Comment:
TypeScript major update addresses errors from `node_modules` declaration files that started showing up after Storybook [`v6.4.0`](https://github.com/storybookjs/storybook/releases/tag/v6.4.0-rc.0). TypeScript's `--skipLibCheck` flag doesn't ignore files from the errors.